numpydoc/tests/test_docscrape.py

```diff
@@ -199,20 +201,25 @@ def test_other_parameters():
 
 
 def test_returns():
-    assert len(doc['Returns']) == 3
+    assert len(doc['Returns']) == 4
     arg, arg_type, desc = doc['Returns'][0]
     assert arg == 'out'
     assert arg_type == 'ndarray'
     assert desc[0].startswith('The drawn samples')
     assert desc[-1].endswith('distribution.')
-
+    
     arg, arg_type, desc = doc['Returns'][1]
+    assert arg == 'dtype'
+    assert arg_type == 'data type object, optional (default : float)'
```

Comment:
This is a bit weird in Returns, rather than Parameters!

But more of an issue: please add a comment to make it clear why we need this test case

Reply:
Oh, yeah, i did not even pay attention to that as the format is the same.
I'll ass a comment and try to move it to parameters.

Reply:
done.